Core/include/Acts/Propagator/MultiEigenStepperLoop.hpp

```diff
@@ -140,74 +144,96 @@ struct WeightedComponentReducerLoop {
   }
 };
 
-struct MaxMomentumReducerLoop {
```

Comment:
this is then typedef'd to `MaxMomentumReducerLoop` right? is this breaking?

Reply:
Na I think exchanging an actual class to a typedef that has the same semantics is not breaking, right?